tests/cpp/cell/test_copy.cu

```diff
@@ -1,9 +1,95 @@
-#include "tiled_cuda_test.h"
+#include "cell/copy/dyn_copy.hpp"
+#include "cell/sync.hpp"
+#include "cell/traits/copy.hpp"
+#include "common/test_utils.hpp"
+
+#include <thrust/device_vector.h>
+#include <thrust/host_vector.h>
+
+namespace traits = tiledcuda::cell::traits;
 
 namespace tiledcuda {
+
+namespace {
+// the host function to call the device copy function
+template <typename Element, typename G2STraits, typename S2GTraits>
+__global__ void Copy(const Element* src, Element* trg) {
```

Comment:
Using underscore naming seems like a better choice.

Reply:
I remove the anonymous namespace.

Reply:
I'm not sure if using PascalCase is appropriate, and I'm a bit confused about the naming when building this repository. Currently, all kernel-related names are in underscore_case, while CUDA utils-related function names use PascalCase. Is it necessary to unify them?

Reply:
Ohhh. I agree lower case should be used. I just didn't notice this issue. Fix it.